src/pages/ChatPage/components/CallRequestDialog.tsx

```
@@ -0,0 +1,70 @@
+  onCancel,
+  text,
+}: CallRequestDialogProps) {
+  const { t } = useTranslation("chat");
+
+  if (!isOpen) return null;
+
+  return (
+    <div
+      className="fixed inset-0 z-50 flex items-center justify-center bg-black/60 backdrop-blur-sm transition-opacity duration-200"
+      onClick={onCancel}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)